appsec-kit-v8/src/main/java/com/vaadin/appsec/v8/ui/content/VulnerabilityDetailsView.java

```diff
@@ -218,21 +228,72 @@ private Component buildVaadinAnalysis() {
         vaTitle.setWidth(100, Unit.PERCENTAGE);
         vAnalysisPanel.addComponent(vaTitle);
 
-        // TODO Replace with real data when available
-        Label vaStatus = new Label("Under review");
-        vaStatus.addStyleName("va-under-review");
+        boolean isSupportedVaadinVersion = false;
+        String vaadinVersion = Version.getFullVersion();
+        AppSecService appsec = AppSecService.getInstance();
+        List<String> fw7Versions = appsec.getSupportedFramework7Versions();
```

Comment:
Why do we need to check for the 7 versions in the `appsec-kit-v8`?

Reply:
Removed also here.